packages/core/src/editor/BlockNoteEditor.ts

```diff
@@ -180,6 +182,8 @@ export class BlockNoteEditor<
 
   public readonly uploadFile: ((file: File) => Promise<string>) | undefined;
 
+  public trailingBlock?: boolean = false;
```

Comment:
Can we remove this member, and just pass the option to `getBlockNoteExtensions` on line 249, similar to the other options?

Reply:
Yes, it's better and resulted in less code. Thanks! 👍